src/modules/wallet/components/RegisterMultisigForm/CategorySwitch.js

```diff
@@ -1,39 +1,28 @@
 import React from 'react';
-import { useTranslation } from 'react-i18next';
 import styles from './styles.css';
 
-const CategorySwitch = ({ changeCategory, isMandatory, index }) => {
-  const { t } = useTranslation();
-  return (
-    <div className={styles.categoryWrapper} onChange={changeCategory}>
-      <input
-        type="radio"
-        id={`mandatory-${index}`}
-        name={`member-category-${index}`}
-        value="mandatory"
-        defaultChecked={isMandatory}
-      />
-      <label
-        htmlFor={`mandatory-${index}`}
-        className={`mandatory select-mandatory ${styles.memberCategory} mandatory-toggle`}
-      >
-        {t('Mandatory')}
-      </label>
-      <input
-        type="radio"
-        id={`optional-${index}`}
-        name={`member-category-${index}`}
-        value="optional"
-        defaultChecked={!isMandatory}
-      />
-      <label
-        htmlFor={`optional-${index}`}
-        className={`optional select-optional ${styles.memberCategory} mandatory-toggle`}
-      >
-        {t('Optional')}
-      </label>
-    </div>
-  );
-};
+const CategorySwitch = ({ changeCategory, categories = [], value, index = 0 }) => (
+  <div className={styles.categoryWrapper} onChange={changeCategory}>
```

Comment:
div element doesnt have an `onChange` attribute.

Reply:
Is also warned about in chrome devtools 

<img width="822" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/b2ee56e1-2234-40f3-b7de-46d4829c4d8e">

Reply:
Seems to work though, but we can probably move that to the <input> element instead to get rid of the error